sway/commands/hide_edge_borders.c

```diff
@@ -23,6 +29,7 @@ struct cmd_results *cmd_hide_edge_borders(int argc, char **argv) {
 		config->hide_edge_borders = E_SMART_NO_GAPS;
 	} else {
 		return cmd_results_new(CMD_INVALID, "Expected 'hide_edge_borders "
+				"'[--i3]' "
```

Comment:
The line wrapping is off here and there should not be single quotes around the flag. It looks like the flags does fit on the line below once the single quotes are removed.